roles/openshift_health_checker/callback_plugins/zz_failure_summary.py

```diff
@@ -95,7 +95,7 @@ def _print_check_failure_summary(self, failed_checks, context):
             'Variables can be set in the inventory or passed on the\n'
             'command line using the -e flag to ansible-playbook.\n'
         ).format(playbook=self._playbook_file, checks=checks)
-        if context in ['pre-install', 'health']:
+        if context in ['pre-install', 'pre-upgrade', 'health']:
```

Comment:
@rhcarvalho @juanvallejo I guess a comment would have helped. So what prompted this was that if you're running a playbook like [byo/config.yml](https://github.com/openshift/openshift-ansible/blob/4f129045b3f3c1b22203a7ab68cda15ee4b8d87b/playbooks/byo/openshift-cluster/config.yml#L11) then these checks running may come as a surprise and it needs a little more explanation. If you're running a playbook that's specifically for running checks, it's not about "failing early" as they're the whole point, and so the default preamble above is a little out of place. I could imagine other cases in which we'd like to customize the message or even the outcome based on the user intent/expectations. But for now, there are really just two cases: you're trying to run checks, or you're trying to get something else done and the checks are "in the way".

So a "pre-upgrade" context would mean you're running a playbook specifically to see if your upgrade is gonna work out. An "upgrade" context would mean you're actually trying to run the upgrade.